Thu Aug 24 2006 Stuart Hughes <stuarth@freescale.com>

Here's a minor update to the base html stuff for skell

diff --exclude CVS -uNr skell-1.13/var/www/cgi-bin/test_cgi skell-1.13.modified/var/www/cgi-bin/test_cgi
--- skell-1.13/var/www/cgi-bin/test_cgi	2005-06-12 17:22:04.000000000 +0100
+++ skell-1.13.modified/var/www/cgi-bin/test_cgi	2006-08-24 11:42:45.000000000 +0100
@@ -1,8 +1,8 @@
 #!/bin/sh
 
-echo Content-type: text/html
-echo ""
-cat <<EOF
+echo "Content-type: text/html
+
+
 <html>
 <head>
     <title>CGI test</TITLE>
@@ -10,4 +10,4 @@
 <h1> CGI test example </h1>
 </body>
 </html>
-EOF
+"
Binary files skell-1.13/var/www/html/favicon.ico and skell-1.13.modified/var/www/html/favicon.ico differ
diff --exclude CVS -uNr skell-1.13/var/www/html/index.html skell-1.13.modified/var/www/html/index.html
--- skell-1.13/var/www/html/index.html	1970-01-01 01:00:00.000000000 +0100
+++ skell-1.13.modified/var/www/html/index.html	2006-08-24 15:26:21.000000000 +0100
@@ -0,0 +1,12 @@
+<html>
+<head>
+<title> index.html test page for http server </title> 
+</head>
+<body bgcolor="#FFFFFF">
+<h1 align=center> Test Page </h1>
+There is a CGI test page <a href=/cgi-bin/test_cgi> here </a>
+<br>
+There is a copy of the LTIB FAQ <a href=http://www.bitshrine.org/autodocs/LtibFaq.html> here </a>
+<br>
+</body>
+</html>
diff --exclude CVS -uNr skell-1.13/var/www/index.html skell-1.13.modified/var/www/index.html
--- skell-1.13/var/www/index.html	2005-06-12 17:20:22.000000000 +0100
+++ skell-1.13.modified/var/www/index.html	1970-01-01 01:00:00.000000000 +0100
@@ -1,8 +0,0 @@
-<html>
-<head>
-<title> index.html test page for http server </title> 
-</head>
-<body bgcolor="#FFFFFF">
-<h1 align=center> Test Page </h1>
-</body>
-</html>
